pkg/virt-config/featuregate/active.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package featuregate
+
+const (
```

Comment:
Why do we need all these files, how does it help me to look at 3 different files?

Reply:
What 3 files?

There are the FGs that are active (Alpha or Beta) and the ones that are inactive (GA, deprecated and discontinued).
This is the separation that made sense to me, over having everything dump on one big file.

Reply:
>over having everything dump on one big file.
That is actually much easier to find. The file will no be big